docs/reference/search/suggesters/phrase-suggest.asciidoc

```diff
@@ -213,8 +213,8 @@ The response contains suggestions scored by the most likely spell correction fir
     Checks each suggestion against the specified `query` to prune suggestions
     for which no matching docs exist in the index. The collate query for a
     suggestion is run only on the local shard from which the suggestion has
-    been generated from. The `query` must be specified, and it is run as
-    a <<query-dsl-template-query,`template` query>>.
+    been generated from. The `query` must be specified and it can be templated,
+    see <<search-template,search templates>> for more information.
```

Comment:
This is a very good change because it has been more like the search templates then the template query for a while now...